bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpRendererHandler.java

```diff
@@ -1113,6 +1113,11 @@ private void onValueReceivedVolume(String variable, @Nullable String value) {
             long volume = Long.valueOf(value);
             volume = volume * 100 / config.maxvolume;
 
+            if (volume < 0) {
+                logger.warn("UPnP device {} received invalid volume value {}", thing.getLabel(), value);
```

Comment:
I'm wondering if this could risk spamming the log for such a device? If there is a risk of ending here quite often, I would suggest to either reduce log level to **debug**, or provide logic so that only the first violation is logged and the all subsequent ones are ignored without logging (or only **debug** logging).

Reply:
This would be at start, or when changing volume. I don’t think that is spamming it. There is clearly something wrong and setting volume will not work if we receive this response.

Reply:
S o we keep it as it is?

Reply:
That's my opinion, but I don't have the final say on this.